index.js

```diff
@@ -0,0 +1,45 @@
+var express=require("express");
+var app=express();
+var arr = require('./halls.js');
+var duljina=arr.length;
+
+app.use(express.static('public'));
+app.get("/",function(req,res){
+    res.send("HELLO");
+});
+
+app.use("/halls",function(req,res){
+    res.json(arr);
+});
+app.use(express.urlencoded());
+app.post("/hallsCreate",function (req,res){
+    const formName = req.body.hallName;
+    arr[arr.length]={id:arr.length+1,name:formName,reservation:{isReserved:false,reservedFrom:null,reservedUntil:null}};
```

Comment:
ne koristiti length + 1 jer onda kad se obriše neki objekt, idući koji se kreira bi imao isti id kao i posljednji. Umjesto toga koristiti id od posljednjeg objekta + 1.

Također, funkcije za CRUD tog arraya je bilo bolje da si zadržao u file-u s tim arrayom kao što se radilo u zadaći 6. A potom ih ovdje samo importati i koristiti